tasks/timeConverter.js

```diff
@@ -0,0 +1,22 @@
+const COUNT = 60;
+function fromSecondsToMinutes(seconds){
+    return seconds / COUNT;
+}
+
+function fromSecondsToHours(seconds){
+    return seconds / (COUNT*COUNT);
+}
+function fromHoursToMinutes(hours){
+    return hours * COUNT;
+}
+function fromHoursToSeconds(hours){
+    return hours * (COUNT*COUNT);
+}
+let RandomNumber = getRandomNum(1,100)
+function getRandomNum(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max-min + 1)) + min;
+}
+
```

Comment:
Pls move this function `getRandomNum` to another file because it is not related to timeConverter
We can use it for other goals, it is something common 

And generate random num right in your tests not here